cloudinit/sources/DataSourceAzure.py

```diff
@@ -675,9 +675,12 @@ def crawl_metadata(self):
         return crawled_data
 
     @azure_ds_telemetry_reporter
-    def get_metadata_from_imds(self, retries: int = 10) -> Dict:
+    def get_metadata_from_imds(self) -> Dict:
```

Comment:
Yeah, it's effectively the greater of the two.  The reason the one path was set to 300 was because it was treated as a hard provisioning failure and errors were apparent.  300 retries was long enough to address the tail issues and now we're just regrouping and treating both paths the same (300 retries @ 1 second sleep is in the ballpark of 300 seconds).